packages/database/src/core/util/libs/parser.ts

```diff
@@ -101,7 +101,9 @@ export const parseRepoInfo = function(
       parsedUrl.host,
       parsedUrl.secure,
       namespace,
-      webSocketOnly
+      webSocketOnly,
+      '',
+      parsedUrl.namespace != parsedUrl.subdomain
```

Comment:
When using literals or boolean expressions as function arguments, it's often helpful to comment them for the reader, something like:

```
    webSocketOnly,
    /*persistenceKey=*/ '',
    /*withQueryParam=*/ parsedUrl.namespace != parsedUrl.subdomain
```